ssm/ssm.go

```diff
@@ -31,6 +28,63 @@ func ProcessParameters(svc ssmiface.SSMAPI, parameters map[string]string, enviro
 	}
 }
 
+func CleanParameters(svc ssmiface.SSMAPI, path string, verbose bool, plainParameters map[string]string, encryptedParameters map[string]string) {
```

Comment:
We should make sure that the path is not used as a wildcard, for instance by adding a trailing slash if it is missing. Otherwise unexpected (and potentially dangerous stuff) may happen, like the prefix `/codacy/worker-manager` being deleted when cleaning `/codacy/worker`.

Reply:
Forced a "/" at the end of the provided path, if the path was provided.